settings.gradle.kts

```diff
@@ -44,10 +44,6 @@ include(":examples:multi-module-sample:spring-boot-in-multi-module-sample")
 
 include(":examples:multi-module-sample:spring-boot-in-multi-module-sample2")
 
-includeBuild("plugin-build") {
-  dependencySubstitution {
```

Comment:
hm, is this going to work for the projects under the `examples` dir? I was under the impression if we remove substitution then they'd pull the gradle plugin from remote and not use the local one?

Reply:
This substitution should work automatically. Here is the how it figures out what to substitute : https://docs.gradle.org/current/userguide/composite_builds.html#included_build_declaring_substitutions

Reply:
TIL 👍